projects/canopy/src/lib/promo-card/promo-card-list/promo-card-list-title/promo-card-list-title.component.scss

```diff
@@ -0,0 +1,11 @@
+@import '../../../../styles/mixins';
+
+.lg-promo-card-list-title__heading {
+  margin-top: var(--space-md);
+  margin-bottom: var(--space-lg);
+
+  @include lg-breakpoint('lg') {
```

Comment:
This adds a bit more of a case for the work proposed in https://github.com/Legal-and-General/canopy/issues/110
Then we could do all of this with directives and have one less css file.